fiorella/src/styles/layout/_category-products-list.scss

```diff
@@ -0,0 +1,154 @@
+.category-product-list {
+  height: 100%;
+  padding: 60px 10px;
+  position: relative;
+  width: 100%;
+}
+
+.category-box {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+
+  @media #{ $tablet }, #{ $desktop } {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 90%;
+  }
+}
+
+.category-item {
+  margin: 5px 0px;
+  text-align: center;
+
+  .category-title {
+    display: none;
+    font-family: $playfair-display;
+
+    @media #{ $tablet } {
+      display: block;
+      font-size: 18px;
+    }
+
+    @media #{ $desktop } {
+      display: block;
+      font-size: 22px;
+    }
+  }
+
+  .category-description {
+    display: none;
+    font-family: $merriweathersans;
+    @media #{ $tablet } {
+      display: block;
+      font-size: 16px;
+    }
+
+    @media #{ $desktop } {
+      display: block;
+      font-size: 23px;
+    }
+  }
+
+  @media #{ $tablet }, #{ $desktop } {
+    margin: 0px 15px;
+  }
+}
+
+.category-content-image {
+  align-items: center;
+  display: flex;
+  height: 200px;
+  justify-content: center;
+  position: relative;
+  width: 100%;
+  z-index: 2;
+
+  .category-image-item {
+    background: $cerise-red-hover;
+    height: 100%;
+    object-fit: cover;
+    width: 100%;
+
+    @media #{ $tablet }, #{ $desktop } {
+      height: 100%;
+      width: 100%;
+    }
+  }
+
+  .category-title-mobile {
+    color: $font-color-white;
+    font-family: $playfair-display;
+    font-size: 30px;
+    font-weight: bold;
+    position: absolute;
+    top: 60px;
+    z-index: 1;
+
+    @media #{ $tablet }, #{ $desktop } {
+      display: none;
+    }
+  }
+
+  .overlay {
+    background-color: $cerise-red-hover;
+    bottom: 0px;
+    height: 100%;
+    position: absolute;
+    width: 100%;
+
+    @media #{ $tablet }, #{ $desktop } {
+      display: none;
+    }
+  }
+
+  .btn-shop {
+    display: none;
+    font-family: $playfair-display;
+    font-weight: bold;
+    position: absolute;
+
+    @media #{ $tablet } {
+      @include hover-categories(transparent,$border-color-white, 2px, $font-color-white, 16px, 48px, 166px);
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in mixin arguments should be followed by a single space